media-gfx/sam2p/files/sam2p-configure-strip.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/Makefile b/Makefile
+index 0e86218..635b4b3 100644
+--- a/Makefile
++++ b/Makefile
+@@ -28,12 +28,13 @@ ifeq ($(MAKECMDGOALS),Makedep.force)
+ MAKE_DIST=000
+ endif
+ 
++GCC_STRIP ?= -s
+ BASH=bash
+ PERL_BADLANG=x
+ export PERL_BADLANG
+ 
+ CXXX=$(CXD_assert)
+-LDALL=$(LDXX) -s $(LDFLAGS) $(LIBS)
++LDALL=$(LDXX) $(GCC_STRIP) $(LDFLAGS) $(LIBS)
+ ifeq ($(ENABLE_DEBUG), no)
+ CXXX=$(CXD_no)
+ endif
+@@ -101,8 +102,8 @@ IDE_MODES := release debug
+ IDE_TARGETS_debug := $(patsubst %,%.yes, $(TARGETS))
+ IDE_TARGETS_release := $(TARGETS)
+ 
+-CXD_assert =$(CXX) -s -O2
+-CXD_no     =$(CXX) -s -DNDEBUG -O3
++CXD_assert =$(CXX) $(GCC_STRIP) -O2
```

Comment:
doesn't this imply you'll be forcing `-O2` down users' throats regardless?

Reply:
Added a fix to the patch, seems like user-configurability of CXXFLAGS was broken since 0.49.3.